examples/libtorchaudio/speech_recognition/greedy_decoder.py

```diff
@@ -0,0 +1,28 @@
+import torch
+
+
+class Decoder(torch.nn.Module):
```

Comment:
I think it would be good to be more specific and name this `GreedyDecoder`, since we may be adding other decoders to torchaudio as well, and the wav2letter pipeline also calls it `GreedyDecoder` [here](https://github.com/pytorch/audio/blob/14dd917ec60fa69ce3f7c6e3f2eaf520e67928b5/examples/pipeline_wav2letter/ctc_decoders.py)

Reply:
So the reason why this is simply called `Decoder` is that in `build_pipeline_....py`, there are components `Loader`/`Decoder`/`Encoder` to illustrate the ASR components in a simplest manner.
If, in future, torchaudio natively supports a decoder, we will update the implementation of this class to serve as an adopter/guru code of the decoder from torchaudio. The interface will be specialized for the example code.
So I see this as a container class. (and currently the core decoding algorithm is missing so we default to hard-coded greedy algorithm)